PSOperations/OperationQueue.swift

```diff
@@ -96,7 +98,9 @@ open class OperationQueue: Foundation.OperationQueue {
             */
             operation.addCompletionBlock { [weak self, weak operation] in
                 guard let queue = self, let operation = operation else { return }
-                queue.delegate?.operationQueue?(queue, operationDidFinish: operation, withErrors: [])
+                if let delegate = queue.delegate {
```

Comment:
Why write something more verbose? Optional chaining already indicated that we are only performing the delegate if the delegate exists.